.gitignore

```diff
@@ -1,9 +1,13 @@
-env
-.env
-node_modules/
-*.DS_Store
-.cache
-yarn-error.log
-.vscode
-dist/
-base-sprite.svg
\ No newline at end of file
+# Logs
+logs
+*.log
+npm-debug.log*
+yarn-debug.log*
+yarn-error.log*
+
+# Other
+node_modules
+.DS_Store
+dist
+.editorconfig
+.vscode/
```

Comment:
Extraordinarily nitpicky, but `.vscode/` is probably better suited for `.git/info/exclude` since it's user-, not project-, specific.